### DISCUSSION TITLE
Xdrip + Freestyle Libre 2 + Wear Os via bluetooth

### DISCUSSION THREAD

**Discussion by columbiecf1** [NightscoutFoundation/xDrip#2827] [General]
Hello,

i would like to ask if there is any possibility of direct bluetooth connection between Xdrip + Wear Os + Freestyle Libre 2?
Until recently, I only used Xdrip + Freestyle Libre 1 + Miaomiao in conjunction with a smartwatch. It was great because I do sports (running, swimming, ...) and I didn't have to have my phone with me.
I searched for options but couldn't find any reasonable solution.

Thanks everyone for any help.

**Comment by PhilTU** [comment]
I think only with transmitter (MM) because the watch does not have NFC functionality to initiate direct connection to sensor.

At least one other app has managed to find a way of transferring the key from phone to watch, but not xDrip as yet.

**Reply by columbiecf1** [reply]
Some smart watches have NFC. My watch has it. It's a shame. Because that would be great!

**Reply by PhilTU** [reply]
it's not so much that it exists, but whether WearOS gives access to it to apps other than Google Pay / Wallet

https://issuetracker.google.com/issues/213599626?pli=1

**Comment by rafass3003** [comment]
@columbiecf1 , could you find out a way to connect the watch to Libre 2 (via xdrip)?

**Comment by columbiecf1** [comment]
Hi, I'm sorry but no.

---------- Původní e-mail ----------
Od: rafass3003 ***@***.***>
Komu: NightscoutFoundation/xDrip ***@***.***>
Datum: 18.06.2024 01:56:03
Předmět: Re: [NightscoutFoundation/xDrip] Xdrip + Freestyle Libre 2 + Wear 
Os
via bluetooth (Discussion #2827)

"



@columbiecf1(https://github.com/columbiecf1) , could you find out a way to 
connect the watch to Libre 2 (via xdrip)?



—
Reply to this email directly, view it on GitHub
(https://github.com/NightscoutFoundation/xDrip/discussions/2827#discussioncomment-9799874)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/A6TAHGG7M5DQOR3MCYOQQ6TZH5ZQ7AVCNFSM6AAAAABJO5Z5USVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOJZHA3TI)
.
You are receiving this because you were mentioned. Message ID: <
***@***.***
>

"

**Reply by Navid200** [reply]
By responding to an email in order to post, you make the thread harder to follow for some.
This is a GitHub problem that they include in the email notification that you can respond to the email without providing a way to avoid quoting the entire post you are responding to.
